README.md

```diff
@@ -100,28 +100,30 @@ wget -P <path to this repository>/data https://rapids-single-cell-examples.s3.us
 
 ### Example Code
 
-Follow this [Jupyter notebook](notebooks/1M_brain_gpu_analysis_uvm.ipynb) for RAPIDS analysis of this dataset. In order for the notebook to run, the files [rapids_scanpy_funcs.py](notebooks/rapids_scanpy_funcs.py) and [utils.py](notebooks/utils.py) need to be in the same folder as the notebook. This notebook runs completely in under 15 minutes on a Tesla V100 GPU with 32 GB memory.
+Follow this [Jupyter notebook](notebooks/1M_brain_gpu_analysis_uvm.ipynb) for RAPIDS analysis of this dataset. In order for the notebook to run, the files [rapids_scanpy_funcs.py](notebooks/rapids_scanpy_funcs.py) and [utils.py](notebooks/utils.py) need to be in the same folder as the notebook. This notebook runs completely in under 15 minutes on a Tesla V100 GPU with 32 GB memory on a DGX with 256GB of main memory.
```

Comment:
I think we can delete the sentence "This notebook runs completely in under 15 minutes on a Tesla V100 GPU with 32 GB memory on a DGX with 256GB of main memory." Now that we have examples of AWS and GCP instances and how long it takes on them.

Reply:
What do you think about replacing the DGX sentence with a small note/disclaimer about the the host memory requirement? I'm mostly just concerned with the surprise when users realize 64gb of main memory is not enough with a 16gb GPU.

Reply:
Yes, I think that would be good.